docs/getting-started/examples.rst

```diff
@@ -14,6 +14,7 @@ Wagtail docs:
 .. code-block:: python
```

Comment:
`GraphQLRichText` needs documenting in `docs/general-usage/model-types.rst`

Reply:
Updated in https://github.com/torchbox/wagtail-grapple/pull/266/commits/763c244a8b659337f077672a1cb6c51ee70d6b98